documentation/docs/pages/Parameter-choices.md

```diff
@@ -0,0 +1,249 @@
+
+### Surface freezing and salinity constraints
+At the ocean surface, we have enabled the realistic sea-ice formation and salinity. We've turned on frazil ice formation (`FRAZIL = TRUE`), which means if the ocean surface layer temperature drops below the surface freezing point, the model will convert the negative heat content into frazil ice instead of allowing the water to supercool. By enabling frazil, any heat deficit below freezing in the ocean is passed to the sea-ice component as latent heat of fusion, preventing unrealistically cold liquid water. 
+
+We also ensure salinity never goes negative by setting `BOUND_SALINITY = True`. In coupled models, when sea ice forms or melts, large salinity fluxes occur. This switch prevents numerical issues when the top model cell might be depleted of salt by excessive ice formation. With this parameter on, salinity is clipped at a minimum value (`MIN_SALINITY = 0.0`). If the sea-ice model were to extract more salt than available, the ocean salinity will be floored at 0 and any deficit is ignored, avoiding unphysical negative salinity. In practice, this situation is rare, but it is a safety measure against extreme scenarios (eg, rapid freezing in nearly fresh water). We also set `SALINITY_UNDERFLOW = 0.0`, meaning any tiny salinity values (close to 0) are reset to exactly 0 as well.
+
+Another parameter we adjust is `HFREEZE = 10.0`. This means the model computes a "melt potential" over a `10m` layer for sea-ice melt/freeze processes. If `HFREEZE >0`, the ocean will calculate how much heat is available in the top 10 meters to melt ice. 
+
+### Surface salinity restoring
+Sea surface salinity is restored to a reference field (`RESTORE_SALINITY = True`, `SALT_RESTORE_FILE = "salt_sfc_restore.nc"`) with a piston velocity of 0.11 m/day (`FLUXCONST = 0.11`). The restoring is implemented as a virtual salt flux ( `SRESTORE_AS_SFLUX = True`). This approach conserves salt overall (balanced globally by subtracting the mean flux, because we set `ADJUST_NET_SRESTORE_TO_ZERO = True` to avoid altering global salinity). No effective limit is applied to the salinity restoring applied (`MAX_DELTA_SRESTORE = 999`). More discussions and decisions can be found at [issues/350](https://github.com/ACCESS-NRI/access-om3-configs/issues/350), [issues/325](https://github.com/ACCESS-NRI/access-om3-configs/issues/325), [issues/257](https://github.com/ACCESS-NRI/access-om3-configs/issues/257).
+
```

Comment:
The salt restoring file is monthly climatological sea surface salinity obtained from World Ocean Atlas 2023.